data/paper.org

```diff
@@ -114,10 +114,10 @@ The result is that:
 #+ATTR_LATEX: :placement [H]
 [[./paper_figure_1.png]]
 
-If you invest in a meme that has 3 initial upvotes, it needs to earn approximately 50
+If you invest in a meme that has 3 initial upvotes, it needs to earn approximately 90
 additional upvotes before maturity for you to break even. If it gains
-approximately 75 upvotes before maturity, you'll double your money. If it gains
-at least 200, your return will be 450%.
+approximately 132 upvotes before maturity, you'll double your money. If it gains
+at least 200, your return will be \sim260%.
```

Comment:
Should be 160% instead of 260%. When we talk about return percentages, it's usually where 100% gain means doubled your money. Hence, return factor of 2.6 --> gain of 160%.

Reply:
Oops! yes.